app/src/main/res/values/dimens.xml

```diff
@@ -7,6 +7,7 @@
     <dimen name="space_12dp">12dp</dimen>
     <dimen name="space_16dp">16dp</dimen>
     <dimen name="space_24dp">24dp</dimen>
+    <dimen name="space_32dp">32dp</dimen>
```

Comment:
It may be.
In my opinion, I think that there is no problem if the size is `32dp` when layout of all pages is not finished.
But I will leave the final decision!